test/internet/test-dgram-broadcast-multi-process.js

```diff
@@ -153,7 +153,8 @@ if (process.argv[2] !== 'child') {
               assert.strictEqual(
                 count,
                 messages.length,
-                'A worker received an invalid multicast message'
+                'A worker received an invalid multicast message. ' +
+                `Received ${count} and wanted ${messages.length}.`
```

Comment:
@nikkistonge sorry to pester, but I think the message is still misleading.

If you look at the code, the above assertion is run only when **all** workers have received **all** the messages. After that the test validates that each message received by the workers matches the original using a counter for each worker. If the count does not match the assertion throws.

My point is that the "Received x and wanted y." feedback is wrong as all messages have been received but one ore more of them is invalid/corrupted.

If you could just refactor this into `assert.strictEqual(count, messages.length)` that would be 10/10 imho as this will remove any possible confusion by using the default error message.

Thanks.

Reply:
Ok changed, now it will use default message :)